ember/app/ui/profile/index/controller.js

```diff
@@ -0,0 +1,31 @@
+import Controller from "@ember/controller";
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+import { lastValue, restartableTask } from "ember-concurrency-decorators";
+
+export default class ProfileIndexController extends Controller {
+  queryParams = ["activeTab"];
+  @tracked activeTab = 0;
```

Comment:
I personally would use labels here like `"memberships"` or `"identity"` so the code and the url are a bit more verbose.

Reply:
Changed

Reply:
Hmm i dont see any new changes. Did you already push them?

Reply:
Weird i see them locally but there are no new changes

Reply:
What is the reason for not just using the labels in code? IMO `@tracked activeTab = "identity"` is easier to read than with the 1 / 0. I will resolve this, would be interested in your answer though.